tests/python_tests/samples/test_greedy_causal_lm.py

```diff
@@ -33,3 +33,42 @@ def test_sample_greedy_causal_lm(self, convert_model, sample_args):
 
         # Compare results
         assert py_result.stdout == cpp_result.stdout, f"Results should match"
+    
+    @pytest.mark.llm
+    @pytest.mark.samples
+    @pytest.mark.parametrize("convert_model", ["phi-1_5", "TinyLlama-1.1B-Chat-v1.0"], indirect=True)
+    @pytest.mark.parametrize("sample_args", ["Alan Turing was a"])
+    def test_sample_greedy_causal_lm_refs(self, request, convert_model, sample_args):
```

Comment:
should we just extend `test_sample_greedy_causal_lm` to check references via HF?

Reply:
combined tests